content/docs/rfcs/14/README.md

```diff
@@ -161,8 +164,9 @@ pubsub_topic = "/waku/2/default-waku/proto" (0x2f77616b752f322f64656661756c742d7
 message.payload = 0x010203045445535405060708
 message.content_topic = "/waku/2/default-content/proto" (0x2f77616b752f322f64656661756c742d636f6e74656e742f70726f746f)
 message.meta = <not-present>
```

Comment:
Since the timestamp is still optional (IIUC), should we have an example without `timestamp` similar to this example with empty `meta`?

Reply:
so basically timestamp if not explicitly provided is filled anyway by our `fakeWakuMessage` proc. Are you saying to write a test case where we overwrite the filled timestamp as empty?

Reply:
I just remembered, `timestamp` attribute is not null in DB schema https://github.com/waku-org/nwaku/blob/master/waku/waku_archive/driver/sqlite_driver/queries.nim#L72 WDYT @vpavlin